cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -82,7 +81,11 @@ def test_vm_mac_scvmm():
             1.
             2. MAC Address should match what is in SCVMM
     """
-    pass
+    collection = provider.appliance.provider_based_collection(provider)
+    vm = collection.all()[0]
+    # compare mac_address in CFME to that from SCVMM
+    mac_addresses = [entry["PhysicalAddress"] for entry in vm.mgmt.raw["VirtualNetworkAdapters"]]
+    assert vm.mac_address in mac_addresses
```

Comment:
There is an inherit assumption here being made, that the `mac_address` property is supplied via a UI lookup.

Reading the original testSteps, its intended that the address from the UI is compared to what's in SCVMM itself.

I think it would be better to directly navigate to the `vm` 'Details' page and read its `SummaryTable` for the MAC, then look for _that_ within the `mac_addresses` list.

A TODO here with an issue in wrapanapi to support getting the MAC for specific VM's would be great. That way we don't have this loose check, and can positively assert that a specific VM's MAC address is mapped correctly.

Reply:
@mshriver that is because of https://github.com/ManageIQ/integration_tests/blob/f0dd9b215a687cb75c8b939ab3108403c05eec3e/cfme/common/vm.py#L191, i.e. the mac_address is obtained via a UI lookup already. And then we are looking for that in the mac_addresses list (which is obtained from SCVMM). Given the `TODO` of the `mac_address` property in the `cfme/common/vm.py`, I can just explicitly do what that function does here, to avoid having to do it later when the `TODO` is fulfilled.